src/lib.ts

```diff
@@ -128,23 +128,17 @@ export async function TransformUrl(url: string, isLocal: boolean) {
     try {
       const questionMatch = patterns.question.exec(urlString);
       if (questionMatch) {
-        return isLocal
-          ? `http://localhost:8787/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`
-          : `https://www.fxzhihu.com/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`;
+        return (isLocal ? `http://localhost:8787` : `https://www.fxzhihu.com`) + `/question/${questionMatch.pathname.groups.questionId}/answer/${questionMatch.pathname.groups.answerId}`
```

Comment:
Is it possible to get the URL from `Env`

Reply:
fixed